vllm/v1/engine/async_llm.py

```diff
@@ -214,6 +214,14 @@ async def generate(
                 # task switching under load which helps performance).
                 out = q.get_nowait() if not q.empty() else await q.get()
 
+                # Coalesce any additional queued outputs
+                while not q.empty():
+                    next_out = q.get_nowait()
+                    if sampling_params.output_kind == RequestOutputKind.DELTA:
```

Comment:
Can be in a future PR, but we should check for invalid sampling params like `n > 1` here

Reply:
Yes I guess that's a general V1 thing, we should be explicitly rejecting requests that include unsupported parameters if we aren't already.